src/services/userService.js

```diff
@@ -0,0 +1,130 @@
+import {Op} from 'sequelize';
+import dotenv from 'dotenv';
+import UserSchema from '../models/userSchema';
+import bcrypt from 'bcrypt-promise';
+import ErrorHandler from '../errors/errorHandler.js';
+
+dotenv.config();
+
+export default class UserService {
+    constructor(userModel) {
+        this.userModel = userModel;
+    }
+
+    async findAutoSuggest(loginSubstring, limit) {
```

Comment:
It would be better to use either promises or async functions for all application